src/app/favlist/layout.tsx

```diff
@@ -7,7 +8,7 @@ export const metadata: Metadata = {
 };
 
 const FavListLayout = ({ children }: { children: React.ReactNode }) => {
-  return <>{children}</>;
+  return <TrpcProvider>{children}</TrpcProvider>;
```

Comment:
I'm wondering if we need `<TrpcProvider>` here as we already have it at root layout.